code/modules/mob/living/carbon/human/species_types/furry/tabaxi.dm

```diff
@@ -37,8 +37,8 @@
 		OFFSET_NECK_F = list(0,-1), OFFSET_MOUTH_F = list(0,-1), OFFSET_PANTS_F = list(0,0), \
 		OFFSET_SHIRT_F = list(0,0), OFFSET_ARMOR_F = list(0,0), OFFSET_UNDIES_F = list(0,0)
 		)
-	specstats = list("strength" = 0, "perception" = 1, "intelligence" = -1, "constitution" = 0, "endurance" = 1, "speed" = -1, "fortune" = 0)
-	specstats_f = list("strength" = -1, "perception" = 0, "intelligence" = 2, "constitution" = -1, "endurance" = 0, "speed" = 1, "fortune" = 0)
+	specstats = list("strength" = 0, "perception" = 1, "intelligence" = -1, "constitution" = -1, "endurance" = 0, "speed" = 1, "fortune" = 1)
+	specstats_f = list("strength" = 0, "perception" = 1, "intelligence" = -1, "constitution" = -1, "endurance" = 0, "speed" = 1, "fortune" = 1)
```

Comment:
getting +1 fortune is unneeded since Tabaxi arent favored by any gods or anything. maybe give them +2 speed instead